src/components/PersonLink.tsx

```diff
@@ -0,0 +1,141 @@
+import { Link, useLocation } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+import { GetPersonDates } from '../services/getPersonDates';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, sex, born, died, slug, motherName, fatherName } = person;
+
+  const { pathname } = useLocation();
+
+  const onlySlug = pathname.split('/').slice(2).join();
+
+  const mother = person.motherName ? person.motherName : '-';
+  const father = person.fatherName ? person.fatherName : '-';
+
+  const motherDates = GetPersonDates(motherName, 'mother');
+  const fatherDates = GetPersonDates(fatherName, 'father');
```

Comment:
The `GetPersonDates` function is called with `motherName` and `fatherName`, but these variables are not defined in the current scope. Ensure that you are passing the correct arguments to this function.